tests/test_lane.py

```diff
@@ -449,34 +437,51 @@ def test_apply(self):
         low_quality.quality = 0
         low_quality.random = 1
 
+        self.add_to_materialized_view(
+            [high_quality_low_random, high_quality_high_random,
+             low_quality]
+        )
+
         facets = FeaturedFacets(0.5, False)
-        base_query = self._db.query(Work).join(Work.license_pools)
+        from model import MaterializedWorkWithGenre as work_model
+        base_query = self._db.query(work_model).join(work_model.license_pool)
+
+        def expect(works, qu):
+            expect_ids = [x.works_id for x in qu]
+            actual_ids = [x.id for x in works]
```

Comment:
The variable naming here is swapped.

Reply:
I've fixed this in my worklist-featured-feed branch.